src/middlewared/middlewared/alert/base.py

```diff
@@ -271,15 +271,6 @@ def formatted(self):
         try:
             return self.klass.format(self.args)
         except Exception:
-            logger.debug("Alert class %r was unable to format args %r, falling back to default formatter",
-                         self.klass, self.args, exc_info=True)
-
-            if self.args:
-                try:
-                    return self.text % (tuple(self.args) if isinstance(self.args, list) else self.args)
```

Comment:
This can be removed because default  `self.klass.format` implementation does the same already (and it is not overridden by any existing alert class).